Dockerfile

```diff
@@ -9,6 +9,9 @@ COPY ./package.json yarn.lock ./
 ENV PYTHON=python3.9
 ENV PY_VERSION=3.9.7
 
+RUN sed -i 's|deb.debian.org/debian|archive.debian.org/debian|g' /etc/apt/sources.list && \
```

Comment:
Why do we need that? Is the Dockerfile image version not correct anymore? Is this specific to node16 image?